.github/uffizzi/docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export ENV APP_CONFIG_app_baseUrl=$UFFIZZI_URL
+export ENV APP_CONFIG_backend_baseUrl=$UFFIZZI_URL
```

Comment:
This file should be able to be removed entirely. The exports are not set correctly and not used in practise as the config uses `$UFFIZZI_URL` directly. You can move the start command to the dockerfile itself.

Reply:
Removed it and moved the `node` command to the Dockerfile.

Reply:
I had to put these exports back in because backstage wasn't connecting to the backend without them.